docs/samples/drift-detection/alibi-detect/cifar10/cifar10cd.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: serving.knative.dev/v1
```

Comment:
are we going to integrate this onto inference service spec? Knative Eventing is promoting to v1beta1 now, maybe it is a good timing to have kfserving optionally depend on KNative eventing if user want features like this relying on eventing.

Reply:
Yes this is optional for this demo only

Reply:
Yes we could integrate at present its every light touch as they are asynchronous tasks so as long as you have payload logging active the rest is separate. But for "Gitops" to keep everything together as your outlier detector may be trained on your model it might be good to bring into CRD.